arelle/FileSource.py

```diff
@@ -402,7 +413,20 @@ def taxonomyPackageMetadataFiles(self) -> list[str]:
         for f in (self.dir or []):
             if f.endswith("/META-INF/taxonomyPackage.xml"): # must be in a sub directory in the zip
                 return [f]  # standard package
-        return [f for f in (self.dir or []) if os.path.split(f)[-1] in TAXONOMY_PACKAGE_FILE_NAMES]
+        return [f for f in (self.dir or []) if os.path.split(f)[-1] in PackageManager.TAXONOMY_PACKAGE_FILE_NAMES]
+
+    @property
+    def isReportPackage(self) -> bool:
+        return self.isZip and (
+            any(PackageManager.reportPackageDirPattern.match(f) for f in (self.dir or ()))
+            or os.path.splitext(self.baseurl)[1] in (".xbri", ".xbr"))
+
+    @property
+    def reportPackageFile(self) -> str | None:
+        for f in (self.dir or []):
+            if f.endswith("/META-INF/reportPackage.json"):
```

Comment:
This seems to wrongly identify the entry`/dir/subdir/META-INF/reportPackage.json` as a report package file.